modules/ROOT/pages/tgw-attach-arm.adoc

```diff
@@ -79,13 +79,13 @@ The *Add transit gateway* page lists the steps to create a resource share in AWS
 image::tgw-add-overview.png[Add transit gateway page: Steps to create a resource share]
 +
 [NOTE]
-The MuleSoft AWS account ID that appears depends on your platform (US or EU).
+The MuleSoft AWS Account ID relevant for your Anypoint VPC is displayed at this step and it varies depending on your platform (US or EU). If the AWS configuration is performed by other teams or resources within your organization, please share this MuleSoft AWS account ID with them.
```

Comment:
could we please remove "depending on your platform (US or EU)"

Reply:
Maybe we do: "The MuleSoft AWS Account ID relevant for your Anypoint VPC is displayed at this step and it varies, so please use the Account ID displayed in this step for your AWS configuration. If the AWS configuration is performed by other teams or resources within your organization, please share this MuleSoft AWS account ID with them."